src/components/BookForm.js

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
 function BookForm() {
+  const dispatch = useDispatch();
+
+  const handleAddBook = (e) => {
+    e.preventDefault();
+    const title = e.target.title.value;
+    const author = e.target.author.value;
```

Comment:
- [ ] Kindly notice that you use a local state with `title` and `author` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md#:~:text=Set%20the%20data,to%20an%20object.).